src/Redux/actionCreators/ServiceDetailsActions.js

```diff
@@ -13,8 +14,14 @@ export const UPDATE_FREE_CALLS_REMAINING = "UPDATE_FREE_CALLS_REMAINING";
 const resetServiceDetails = dispatch => {
   dispatch({ type: RESET_SERVICE_DETAILS });
 };
+
+const fetchServiceDetailsFailure = err => dispatch => {
+  dispatch(loaderActions.stopAppLoader);
```

Comment:
You are not using the error message here? Should we show some error message if this fails?